### DISCUSSION TITLE
Achieving equal-length lines in Stepper

### DISCUSSION THREAD

**Discussion by kcpru** [chakra-ui/chakra-ui#7763] [General]
Currently, the line length in the Stepper component depends on the length of the StepTitle and StepDescription, leading to uneven line lengths in certain cases and affecting the overall aesthetics, as illustrated in the example below.

![image](https://github.com/chakra-ui/chakra-ui/assets/47855645/e190a94c-f9eb-4430-8866-f2dfbfad1085)

I would like to inquire if it is possible to configure the Stepper component to ensure that the lines are of equal length regardless of the length of StepTitle and StepDescription. This enhancement would significantly improve the visual consistency and appearance of the stepper.

```jsx
{steps.map(({ icon, title, description }, index) => (
    <Step key={index}>
        <StepIndicator>
          <StepStatus
              complete={<StepIcon />}
              incomplete={<icon />}
              active={<icon />}
            />
        </StepIndicator>

        <Box flexShrink="0" display={{ base: 'none', md: 'block' }}>
            <StepTitle>{title}</StepTitle>
            <StepDescription>{description}</StepDescription>
        </Box>

        <StepSeparator />
    </Step>
))}
```

**Comment by RHEDeclan** [comment]
Anyone have a nice solution for this?
